整！

NanoRocky opened the pull request:
进行亿些小小的功能改动~
------
·【ADD】 增加逐字歌词模块。允许通过 MetingAPI 和 AMLL TTML Database 获得逐字歌词，兼容网易云和QQ音乐。MetingAPI 需要使用修改过的版本， injahow 那边大概是因为我动了上游的 Meting 叭，并没有处理 pr ，Meting 那边也没有动静。使用之前的 MetingAPI 可正常解析网易云音乐逐行歌词，AMLL TTML Database 也会补充部分音乐的逐字。AMLL TTML Database 的调用方式是拿歌曲 id 去获取文件。目前还没有找到什么办法能屏蔽掉浏览器控制台内的 404 提示，包括写一个 js 专门用来获取文件。望其他大佬指点！
·【FIX】依赖组件更新，更新或替代已弃用的依赖，使用新的 global.scss 引用方式，消除依赖弃用提示。
·【ADD】加了一个没球用的语音提示模块，在弹出某些提示时播放语音提示，包含预生成（固定的通知提醒）和实时生成（播报歌名）。
·【ADD】在 Footer 顶边框上加了个音乐显示进度条，在未播放音乐时默认隐藏。允许拖动 Icon 更改进度，现允许双击 Footer 常驻 Icon。
·【ADD】添加了个键盘监听，用来切歌，并设置仅当页面在最前有效。
·【ADD】向浏览器汇报 Media Session 元数据，让 Windows 10 or 11 能正确显示播放控件，并可正常切歌。
·【FIX】修复了一些小 BUG ，如 TimeCapsule.vue 中的 Number(item.percentage) 。
·【FIX】补全漏的 “-webkit” ，增强兼容性。
·【ADD】加了个“季节效果”，在固定月份挂载 js 来添加不同的效果。
·【ADD】为歌词刷新添加渐隐和整行下沉动画效果替代生硬消失效果。
·【ADD】添加独立的 json 用于配置壁纸个数，添加壁纸时无需重新编译项目。
·【ADD】允许在移动端点击 “网站列表” 打开移动端设置。
·【ADD】允许在移动端使用不同的壁纸。
·【FIX】以韩小韩API替代失效的教书先生API
·【ADD】添加腾讯位置服务支持，以解决高德接口次数不足以及不支持 IPV6 的问题
·【ADD】修改天气模块策略，在一个接口失效时切换另一个接口
·【FIX】修复天气模块中的错误，处理气温平均数时自动删除多余符号单位，处理风量时先判断是否携带级字再考虑添加
·【ADD】 使用 POSTCSS 管理 Web 兼容性
......
------
拉着 @pizeroLOL 从暑假整到现在，从 Azure 到 Meting，也算是完成人生中第一个小项目了！（）
对于这堆代码..只能说能跑，嗯...能多shi有多shi。就我写的那充满 warnings 连套六七个 if 的玩意，也算是让 Pizero 改的崩溃了。咳咳，看看有没有其它大佬来修修叭（
目前效果预览：[酪灰の主页](https://nanorocky.top/)

先给作者留一个表情包在这（bushi）：
![地铁老人手机](https://github.com/user-attachments/assets/5b87b438-6a85-40ae-a3b7-f3541e4d63bf)

NanoRocky committed:
beta

NanoRocky committed:
进行一些改进
1.修复 pinia 无法重置变量的问题。
2.修复 weather.vue 中的某些错误。
3.补全遗漏的语音提示。
4.逐字样式修改，添加逐字上色增强效果，但默认不开启，因为这坨山还有 N 个 BUG 要修，仅仅能跑，一点不稳定。
5.修复（？）歌词解析时因 API 返回玄学内容造成的异常。
6.逐字解析逻辑修改。
...
More fixes needed

NanoRocky committed:
函数忘记定义 bug 修复（）

NanoRocky committed:
巧克力好吃，但 pinia 不喜欢

NanoRocky committed:
对网易云返回的令人费解的时间轴为负数等逐字歌词做点措施

NanoRocky committed:
补点参数注释

NanoRocky committed:
BUG 修复，错误的使用 timeupdate 导致函数启动了无限个进程。
Pizero：你在玩函数繁殖吗？

imsyy commented:
好家伙  
![LeonardoDicaprioClappingGIF](https://github.com/user-attachments/assets/491e6aec-ebeb-4822-8e70-1ead9175e416)

NanoRocky committed:
忘记改回去了（）

NanoRocky committed:
逐字模块部分BUG修复，和一些兼容性优化

NanoRocky committed:
Footer 长音效果修复，逐字效果增强，进行一些BUG修复

NanoRocky committed:
弃用依赖替代，逐字样式升级，调用AMLL歌词库默认使用加速镜像源（可在设置内关闭），语音提示添加延迟时间避免洪水

NanoRocky committed:
补了个...键盘切歌的..没用玩意（

NanoRocky committed:
修复api错误，添加季节效果，添加尚未完成的音乐进度条（

pizeroLOL committed:
fix(yrc): 移动位置并改为使用watch实现
又是手欠点了格式化的一天

pizeroLOL committed:
fix(yrc): 关闭 debug flag

NanoRocky committed:
Merge pull request #1 from pizeroLOL/fix/yrc-not-work

fix(yrc): 移动位置并改为使用watch实现

NanoRocky committed:
逐字样式 BUG 修复

NanoRocky committed:
FIX:修复一些BUG.
# 修复错误的 AMLL 歌词获取逻辑
# 修复逐字歌词和逐行歌词的歌词显示延迟问题
# 为歌词刷新添加渐隐和整行下沉动画效果替代生硬消失效果
# 修改默认 Speech 语音生成的等待时间至 1500 毫秒,防止频繁点击产生请求洪水以及消耗生成数

NanoRocky committed:
FIX:修复 AMLL 逐行歌词加载(修了个peach)

NanoRocky committed:
默认禁用逐字库镜像加速，因为镜像站疑似 GFW。虽然已经替换了新的，但为了稳定性不默认开启

dependabot[bot] committed:
build(deps-dev): bump vite in the npm_and_yarn group across 1 directory

Bumps the npm_and_yarn group with 1 update in the / directory: [vite](https://github.com/vitejs/vite/tree/HEAD/packages/vite).


Updates `vite` from 6.0.7 to 6.0.9
- [Release notes](https://github.com/vitejs/vite/releases)
- [Changelog](https://github.com/vitejs/vite/blob/main/packages/vite/CHANGELOG.md)
- [Commits](https://github.com/vitejs/vite/commits/v6.0.9/packages/vite)

---
updated-dependencies:
- dependency-name: vite
  dependency-type: direct:development
  dependency-group: npm_and_yarn
...

Signed-off-by: dependabot[bot] <support@github.com>

NanoRocky committed:
Merge pull request #2 from NanoRocky/dependabot/npm_and_yarn/npm_and_yarn-545022be4d

build(deps-dev): bump vite from 6.0.7 to 6.0.9 in the npm_and_yarn group across 1 directory

NanoRocky committed:
[Add] 添加功能
# 添加独立的壁纸配置，增加壁纸时无需重新编译项目
# 允许为移动端设置不同分辨率的壁纸
# 允许在移动端点击 网站列表  以打开设置
# 依赖更新

NanoRocky committed:
[FIX]修复 Pina 错误缓存所有变量的问题

NanoRocky committed:
天气模块修改
>> ·[FIX]以韩小韩API替代失效的教书先生API
>> ·[ADD]添加腾讯位置服务支持，以解决高德接口次数不足以及不支持 IPV6 的问题
>> ·[ADD]修改天气模块策略，在一个接口失效时切换另一个接口
>> ·[FIX]修复天气模块中的错误，处理气温平均数时自动删除多余符号单位，处理分量时先判断是否携带级字再考虑添加

NanoRocky committed:
[Update]build.yml

pizeroLOL committed:
feat: support progress bar (#7)

* feat: support progress bar

* feat/fix: cleanup event & 修复归位不可用

* feat(MusicProgressBar): support touch device

Signed-off-by: Pizero <zhaory200707@outlook.com>

* feat(MusicProgressBar): 未加载完成时等待加载完成再同步icon位置

---------

Signed-off-by: Pizero <zhaory200707@outlook.com>

NanoRocky committed:
Add and Fix
[FIX] 修复韩小韩天气 API 调用异常。（谁家 API 早晚天气要调用不同函数的，文档也没写几点之后拿晚函数啊...[bushi]）
[FIX] 修复签名函数生成带双斜杠链接的错误
[ADD] 为进度条增加拖动功能，双击 Footer 可固定 Icon
[ADD] 使用 POSTCSS 管理 Web 兼容性
[ADD] 补全语音
[Update] 依赖组件更新

NanoRocky committed:
组件更新

NanoRocky committed:
Merge branch 'dev' into dev

NanoRocky committed:
Merge branch 'dev' into dev
